index.html

```diff
@@ -169,52 +169,12 @@ <h4 class="date">{{event.date}}</h4>
     <section id="committe">
         <div class="container">
             <div class="section-header">
-                <h1 role="section-heading">The committee</h1>
-                <h3>Meet the team behind the workshops and events at HaCS</h3>
-            </div>
-
-            <div class="committee-content">
-              {% for member in site.data.committee %}
-                <div class="committe-item">
-                    <img src="images/team/{{member.name}}.jpg" alt="{{member.name}}"/>
-                    <h2>{{member.name}}</h2>
-                    <h4>{{member.role}}</h4>
-                    <hr/>
-                    <div class="committe-social">
-
-                      {% if member.facebook %}
-                        <a href="https://www.facebook.com/{{member.facebook}}" alt="Facebook"><div class="facebook"></div></a>
-                      {% endif %}
-
-                      {% if member.twitter %}
-                        <a href="https://twitter.com/{{member.twitter}}" alt="Twitter"><div class="twitter"></div></a>
-                      {% endif %}
-
-                      {% if member.github %}
-                        <a href="https://github.com/{{member.github}}" alt="GitHub"><div class="github"></div></a>
-                      {% endif %}
-
-                      {% if member.web %}
-                        <a href="http://{{member.web}}" alt="Website"><div class="web-outline"></div></a>
-                      {% endif %}
-
-                    </div>
-                </div>
-              {% endfor %}
-
-
-        </div>
-    </section>
-
-    <section id="committe">
-        <div class="container">
-            <div class="section-header">
                 <h1 role="section-heading">2017 Committee</h1>
                 <h3>Meet the newly-elected committee of 2017/18!</h3>
             </div>
 
             <div class="committee-content">
-              {% for member in site.data.committee2017 %}
+              {% for member in site.data.committee[2017] %}
```

Comment:
found how to do it @luketainton took some googling but it creeped up in the jekyll docs, I just didnt read them thoroughly enough http://jekyllrb.com/docs/datafiles/